types/elasticsearch/src/lib/connectors/http.d.ts

```diff
@@ -11,3 +11,6 @@ declare class HttpConnector {
     makeReqParams(params: any): any;
     request(params: any, callback: (error: any, response: any, status: any, headers: any) => void): void;
 }
+
+declare namespace HttpConnector {
```

Comment:
By merging a namespace this actually lies to the type system that `HttpConnector` can be imported using `import * as`. Is there an issue with importing using `import =` with Angular 5?

Reply:
I use lastest version of Angular is 5, may be other version from version 2 catch the same error. They use es6 syntax `import ... from ...` while node js use commonjs with `const ... = require(...)`. I read at https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this. I just add one line for angular (I found momentjs also declare too) and node js can work together.

Reply:
I tried with `import =` but it work incorrectly. Look at here `module.exports = HttpConnector;` in `elasticsearchjs` lib, I think this is good solution now, don't you?

Reply:
sorry, wait me re-code, some update from angular 6 and elasticsearch 15 make me confuse.